browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchPersistentStore.java

```diff
@@ -26,15 +31,21 @@ BrowserSwitchRequest getActiveRequest(Context context) {
         if (activeRequestJson != null) {
             try {
                 request = BrowserSwitchRequest.fromJson(activeRequestJson);
-            } catch (JSONException ignored) { /* do nothing */ }
+            } catch (JSONException e) {
+                Log.d(TAG, e.getMessage());
+                Log.d(TAG, Arrays.toString(e.getStackTrace()));
```

Comment:
Take it or leave it question: why logging here instead of having `getActiveRequest` throw an exception?

Reply:
Yeah this was mostly for internal purposes so we can catch errors locally during development. If we throw an exception we would have to bubble up to the merchant at some point via the API.

Since this class is non-public it's easier to just log the message for now and maybe we can re-consider in the future to implement more robust error handling for BrowserSwitch as a whole.